pytest_html/plugin.py

```diff
@@ -273,7 +275,8 @@ def append_extra_html(self, extra, extra_index, test_index):
         def append_log_html(self, report, additional_html):
             log = html.div(class_="log")
             if report.longrepr:
-                for line in report.longreprtext.splitlines():
+                text = report.longreprtext or report.full_text
```

Comment:
Can you add a comment explaining why `longreprtext` is Falsey at times and why we default to `full_text` in those instances?